### PR TITLE
MC-Release-23.1

### PR DESCRIPTION
**Pull request by ishOsw**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-2018
https://tools.hmcts.net/jira/browse/EUI-2283
https://tools.hmcts.net/jira/browse/EUI-3528
https://tools.hmcts.net/jira/browse/EUI-4033
https://tools.hmcts.net/jira/browse/EUI-4112
https://tools.hmcts.net/jira/browse/EUI-4205
https://tools.hmcts.net/jira/browse/EUI-4248
https://tools.hmcts.net/jira/browse/EUI-4268

### Change description ###
Updating ccd-toolkit version

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by paul-graham — COMMENTED** [review]
Is this for testing purposes or with the intention of merging into MC `master`? If the latter, it needs to be pointing at a fully released version of the toolkit, not one with a pre-release suffix... 🤔

File: package.json [hunk 1]
```diff
@@ -69,6 +69,7 @@
     "@angular/platform-browser": "^7.2.13",
     "@angular/platform-browser-dynamic": "^7.2.13",
     "@angular/router": "^7.2.13",
+    "@hmcts/ccd-case-ui-toolkit": "4.1.5-cve-fix",
     "@hmcts/ccd-case-ui-toolkit": "4.1.0-rc.13",
```
Comment (paul-graham) [inline package.json:73]: There are two entries in here for the same thing... 🤔

**Review by paul-graham — CHANGES_REQUESTED** [review]
Two versions of the toolkit specified in `package.json`.

**Review by paul-graham — COMMENTED** [review]
This will need a fully-released version of the CCD Case UI Toolkit... but otherwise, it's fine.

**Review by Mo-Lala84 — APPROVED** [review]
approved

**Review by RiteshHMCTS — DISMISSED** [review]
Approved

**Review by Mo-Lala84 — DISMISSED** [review]
approved

**Review by udaydenduluri33 — DISMISSED** [review]
@ishOsw  @Mo-Lala84  I can see the upgraded CCD-UI toolkit library

**Review by kuda1992 — APPROVED** [review]
Approved with minor code improvement

**Pull request merged**